tests/kafkatest/services/kafka/kafka.py

```diff
@@ -782,7 +782,8 @@ def prop_file(self, node):
 
         if self.use_new_coordinator:
             override_configs[config_property.NEW_GROUP_COORDINATOR_ENABLE] = 'true'
```

Comment:
maybe we can remove this config? `group.coordinator.rebalance.protocols=classic,consumer` has covered that.

Reply:
I would keep it for now. It is nice to have the ability to set it explicitly in tests.

Reply:
> I would keep it for now. It is nice to have the ability to set it explicitly in tests.

that is fine to me. BTW, will we remove `group.coordinator.new.enable` in the future?

Reply:
my point was: why we need the scenario that server uses the new coordinator but "consumer protocol" is disallowed?

Reply:
Yes. We will remove `group.coordinator.new.enable` in the near future. No, we don't have this case at the moment. I think that we have such cases, especially in integration environments where we want to test the new coordinator without the "consumer protocol".